### DISCUSSION TITLE
[General] Unsupported 'application/xml;charset=utf-8' Content-type proxy plugin

### DISCUSSION THREAD

**Discussion by tulhaum** [glpi-project/glpi-agent#460] [General]
### Your question

Hi folks!
Using GLPI-Agent 1.5 with proxy server plugin works fine inventoring computer machines. 
But when I try to inventory a smartphone, Glpi Agent response: "Unsupported 'application/xml;charset=utf-8' Content-type header provided in glpi request from 127.0.0.1"
I can send the smarthphone inventory directly to GLPI server, but fails using the proxy plugin.
Anyone can help or know if is it a bug ?

**Comment by g-bougard** [comment]
Hi @tulhaum 

it was a bug and it will now be fixed in nightly. See #461

**Reply by tulhaum** [reply]
Thank you!
